cdap-cli-tests/src/test/java/co/cask/cdap/cli/CLIMainTest.java

```diff
@@ -354,84 +355,145 @@ public void testDataset() throws Exception {
   public void testService() throws Exception {
     ServiceId service = fakeAppId.service(PrefixedEchoHandler.NAME);
     ServiceId serviceV1 = fakeAppIdV1.service(PrefixedEchoHandler.NAME);
-    String qualifiedServiceId = String.format("%s.%s", FakeApp.NAME, PrefixedEchoHandler.NAME);
-    String qualifiedServiceIdV1 = String.format("%s.%s.%s", FakeApp.NAME, V1_SNAPSHOT, PrefixedEchoHandler.NAME);
-    testCommandOutputContains(cli, "start service " + qualifiedServiceId, "Successfully started service");
-    testCommandOutputContains(cli, "start service " + qualifiedServiceIdV1, "Successfully started service");
-    assertProgramStatus(programClient, service, "RUNNING");
-    assertProgramStatus(programClient, serviceV1, "RUNNING");
+    String serviceName = String.format("%s.%s", FakeApp.NAME, PrefixedEchoHandler.NAME);
+    String serviceArgument = String.format("%s version %s", serviceName, ApplicationId.DEFAULT_VERSION);
```

Comment:
what happens when you don't pass in a version, since version is optional? Should add a test for optional parameters everywhere